django/test/selenium.py

```diff
@@ -0,0 +1,60 @@
+import os
+import types
+from unittest import skipIf
+
+from six import add_metaclass
+
+from django.test import tag
+from django.utils.module_loading import import_string
+from django.utils.six import (
+    get_function_closure, get_function_code, get_function_defaults,
+    get_function_globals,
+)
+
+
+class SeleniumMetaClass(type):
+
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically inject new methods for running tests in different browsers
+        when multiple browser specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        browsers = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')
```

Comment:
What about defining a `browsers` class attribute and make `runtests` set `SeleniumMetaClass.browsers = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')` instead?

This would allow users to either set the browsers to be tested per test class or create a subclass that can be used project wide.